pom.xml

```diff
@@ -15,8 +14,13 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+        <commons.version>1.10</commons.version>
+        <httpcomponents.version>4.5.1</httpcomponents.version>
+        <script-security.version>1.17</script-security.version>
+        <workflow.version>1.13</workflow.version>
```

Comment:
By the way I misspoke earlier; the [current default baseline is 1.625.x](https://github.com/jenkinsci/plugin-pom/blob/plugin-2.3/pom.xml#L50).